src/scripts.js

```diff
@@ -51,13 +52,19 @@ const pendingTrips = document.querySelector('#pendingTrips')
 const tripView = document.querySelector('#tripView')
 const userMenu = document.querySelector('#userMenu')
 const startDate = document.querySelector('#startDate')
+const bottomHeading = document.querySelector('#bottomHeading')
```

Comment:
FYI, since you're selecting all of this by ID, there's a shortcut. You can do - 
`const bottomHeading = document.getElementById('bottomHeading')`

This is a great option when you see yourself naming a class, and an ID the same thing in your HTML, but need to querySelect them for different functions.

Reply:
thank you - making the changes on my local repo!